packages/maskbook/src/plugins/BarnBridge/Model/SYPoolModel.tsx

```diff
@@ -0,0 +1,64 @@
+import { BB_SY_CREAM, BB_SY_AAVE, BB_SY_COMPOUND, APP_URL, SY_URL } from '../constants'
+import { useAsyncRetry } from 'react-use'
+import urlcat from 'urlcat'
+
+export function SmartYieldPoolModelGetData() {
+    const protocolsMap: any = {}
+
+    return useAsyncRetry(async () => {
+        const response = await fetch('https://api.barnbridge.com/api/smartyield/pools?originator=all', {
+            body: null,
+            method: 'GET',
+            mode: 'cors',
+            credentials: 'omit',
+        })
+
+        const realData = await response.json()
```

Comment:
plz provide response payload typing information

```suggestion
        interface Payload<T> {
            data: T
        }
        interface Entry {
            protocolId: string
            underlyingSymbol: string
            state: State
        }
        interface State {
            seniorApy: number
            juniorApy: number
            seniorLiquidity: string
            juniorLiquidity: string
            /** etc ... **/
        }
        const payload: Payload<Entry[]> = await response.json()
```